test/functional/test_framework/key.py

```diff
@@ -356,6 +356,26 @@ def tweak_add(self, tweak):
         ret.compressed = self.compressed
         return ret
 
+    def mul(self, data):
```

Comment:
Add a docstring commenting this function (ie that this multiplies a public key by a scalar).

Reply:
Done.